ruby/protobuf/google/api/annotations.proto

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2015, Google Inc.
```

Comment:
Delete *google* dir from protobuf, please.
You can get google proto files from [third_party](https://github.com/TinkoffCreditSystems/voicekit-examples/tree/master/third_party) for grpc generation.

Reply:
Done